app/jobs/reports/sp_user_counts_report.rb

```diff
@@ -15,7 +15,36 @@ def perform(_date)
       user_counts = transaction_with_timeout do
         Db::Identity::SpUserCounts.call
       end
-      save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
+
+      track_report_data_events(user_counts)
+      results = save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
+
+      track_global_user_total_events
```

Comment:
we can piggy back it here...i could be it's own job

Reply:
the queries are completely independent of the ones for the s3 report right? seems better to be in its own job IMO